core/mvcc/database/mod.rs

```diff
@@ -17,13 +17,25 @@ pub struct RowID {
     pub row_id: u64,
 }
 
+impl RowID {
+    pub fn new(table_id: u64, row_id: u64) -> Self {
+        Self { table_id, row_id }
+    }
+}
+
 #[derive(Clone, Debug, PartialEq, PartialOrd)]
 
 pub struct Row {
     pub id: RowID,
     pub data: Vec<u8>,
 }
 
+impl Row {
+    pub fn new(id: RowID, data: Vec<u8>) -> Self {
```

Comment:
Does it make sense to have data as `Pin<Vec<u8>>`?

Reply:
@pereman2 Not sure. It's not clear to me `Vec<u8>` is the best option either -- perhaps a `Record` is even better. I think we can optimize later.